core/includes/image.inc

```diff
@@ -112,7 +122,9 @@ function image_get_supported_extensions() {
   $toolkit = image_get_toolkit();
   $function = 'image_' . $toolkit . '_supported_extensions';
   if (function_exists($function)) {
-    return $function();
+    $extensions = $function();
+    $extensions[] = 'svg';
```

Comment:
I'm not sure yet, if always adding svg is a good idea... But then - we don't have a condition to apply here. Or do we?

Reply:
`$extensions` array holds all image types that image fields support. The others depend on toolkit support of course. I cant think of any condition to add here either. Either add svg or don't.

Reply:
I think this is okay as-is. If no toolkit needed then it's always supported.